src/shogun/statistical_testing/internals/StreamingDataFetcher.cpp

```diff
@@ -30,14 +30,15 @@ StreamingDataFetcher::StreamingDataFetcher(CStreamingFeatures* samples)
 : DataFetcher(), parser_running(false)
 {
 	REQUIRE(samples!=nullptr, "Samples cannot be null!\n");
```

Comment:
If this can be caused by SWIG interface (i.e. user), should not contain the word NULL